### DISCUSSION TITLE
Emgu.CV.CvInvoke threw an exception problem when deploying in PC

### DISCUSSION THREAD

**Discussion by ogonzalezf** [emgucv/emgucv#785] [Q&A]
Good morning, I am presenting problems of the type: "Emgu.CV.CvInvoke threw an exception", in a C# application on the development PC it does not give me that error and as soon as I pass the executable with the libraries to the test one I get that error and I have already tried to do everything that pose on your site and nothing. Version: Emgu.CV ----4.6

**Comment by sn4k3** [comment]
You always need the runtimes, [Windows](https://www.nuget.org/packages/Emgu.CV.runtime.windows)
